db/structure.sql

```diff
@@ -2228,20 +2226,6 @@ CREATE UNIQUE INDEX index_subject_sets_workflows_on_workflow_id_and_subject_set_
 
 
 --
--- Name: index_subject_workflow_counts_on_set_member_subject_id; Type: INDEX; Schema: public; Owner: -; Tablespace: 
---
-
-CREATE INDEX index_subject_workflow_counts_on_set_member_subject_id ON subject_workflow_counts USING btree (set_member_subject_id);
-
-
---
--- Name: index_subject_workflow_counts_on_sms_id_and_workflow_id; Type: INDEX; Schema: public; Owner: -; Tablespace: 
---
-
-CREATE UNIQUE INDEX index_subject_workflow_counts_on_sms_id_and_workflow_id ON subject_workflow_counts USING btree (set_member_subject_id, workflow_id);
```

Comment:
Seems like there's no index on just `workflow_id`. Probably hasn't been an issue because no subject is used in many workflows (after using the `subject_id` index, we're left with probably up to 2 or 3 rows, and usually only one.